src/components/icons/Lab.tsx

```diff
@@ -4,18 +4,94 @@ const LabIcon: React.FC = () => {
     return (
         <svg
             className="custom-icon"
-            width="12"
-            height="22"
-            viewBox="0 0 12 22"
+            width="10"
+            viewBox="0 -10 40 80"
```

Comment:
just curious, why don't you have the svgs as static file assets?

Reply:
I'm not sure. I think it was because the initial svg I brought in needed to have the color change with the app, so I wanted to send in props, but I could have handled it another way. I wonder if something in the build went wrong with a direct import?